.github/workflows/ci-conform-validator.yml

```diff
@@ -0,0 +1,25 @@
+name: ci-conform-validator
+on:
+  push:
+    branches: [main]
+    paths:
+      - 'packages/conform-validator/**'
+  pull_request:
+    branches: ['*']
+    paths:
+      - 'packages/conform-validator/**'
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./packages/conform-validator
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18.x
```

Comment:
Can you fix it as `20.x` even though some middleware uses `18.x`?

Reply:
OK. I fixed it.

commit: https://github.com/honojs/middleware/pull/666/commits/b345f847bf44375108a37def31c7d3bce415c12f